app/views/users/_user.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= gravatar_for user, size: 50 %>
   <%= link_to user.username, user %>
 
-  <% if current_user.privilege == "admin" && !current_user?(user) %>
+  <% if current_user && current_user.privilege == "admin" && !current_user?(user) %>
```

Comment:
Hmm that's interesting because current_user should never be null in this condition because it's impossible to access the localhost:3000/users without being logged in due to `before_action :logged_in_user, only: [:index,...` in the users_controller.

Can you show me the error later? I'm afraid this might be a bug.